sdk/java/pulumi/src/main/java/com/pulumi/context/internal/ContextInternal.java

```diff
@@ -30,15 +30,14 @@ public ContextInternal(
             String stackName,
             LoggingContextInternal logging,
             ConfigContextInternal config,
-            OutputContextInternal outputs,
-            Map<String, Output<?>> exports
```

Comment:
I've realized that I cannot think of any scenario where this would be pre-set. Please correct me if I'm wrong.

Reply:
I'm not sure... at this point I'm leaning to "if it's not tested it does not exist" but we do need to build up some test coverage for how users interact with this.

Reply:
> "if it's not tested it does not exist"

I like this. Since this is an internal API, this constructor, then it can be easily added back :)